src/index.html

```diff
@@ -15,8 +15,100 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="#home"
+        class="photo"
+      >
+        <img
+          src="Moyo.png"
+          alt="Logo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav_list">
+          <li class="nav_item">
+            <a
+              href="#section-1"
+              class="box box-1 is-active"
+            >
+              APPLE
+            </a>
+          </li>
+          <li class="nav_item">
```

Comment:
Add the blank line between the links
```suggestion
          </li>
          
          <li class="nav_item">
```